pxr/usdImaging/usdviewq/appController.py

```diff
@@ -3189,6 +3204,11 @@ def _toggleShowAbstractPrims(self):
         self._dataModel.selection.removeAbstractPrims()
         self._resetPrimView()
 
+    def _toggleShowPrimDisplayName(self):
+        self._dataModel.viewSettings.showPrimDisplayNames = (
+            self._ui.actionShow_Prim_DisplayName.isChecked())
+        self._resetPrimView()
```

Comment:
Doesn't affect this code, but reminds me to ask for verification: the Prim View browser should already be handling the `signalSettingChanged` signal, and we should make sure that its response includes resetting any search we've previously performed, since its results might differ depending on whether we're showing displayNames.